mk/config.mk

```diff
@@ -38,8 +38,10 @@ WARNS ?= 3
 # Define DEBUG=1 to compile with -g option
 # DEBUG=1
 
-# If 1, debug mode of the tee firmware (CPU restart, Core Status)
-CFG_TEE_CORE_DEBUG ?= 0
+# Core debug mode enablement
+# When defined to "y", tee firmware is in debug mode. Core and pager status,
+# panics verbosity, physical/virtual address paranoid verifications.
+CFG_TEE_CORE_DEBUG ?= n
```

Comment:
question: `CFG_TEE_CORE_DEBUG` was mis used: tested against ifdef and/or ifeq/ifneq 0.
i wanted to rationalize.
but maybe switching from 0/1 to y/n break some backward compatibility ?